source/install/troubleshooting.rst

```diff
@@ -3,60 +3,76 @@
 Troubleshooting
 ===============
 
-This document summarizes common troubleshooting issues and techniques.
+This document summarizes common troubleshooting issues and techniques. 
+
+Depending on the type of error or problem you're experiencing, refer to the sections below for troubleshooting guidance. If you're a new user, it might help to go over the installation steps again to confirm the process. Alternatively, the Troubleshooting forum might be helpful as someone may have experienced the same error in the past. 
+
+For more troubleshooting help, `open a new topic in our forums <https://forum.mattermost.org/c/trouble-shoot>`__ with steps to reproduce your issue. If you're an Enterprise Edition subscriber, you can also email subscribers@mattermost.com for support.
```

Comment:
We could link to Enterprise Edition support page here for details https://about.mattermost.com/support/

PS: I don't think subscribers@mattermost.com is actually used anymore, where did you find it? We may need to update the email address for it.

Reply:
I found the address here: https://docs.mattermost.com/mobile/mobile-troubleshoot.html, in "None of these solve my problem!" so if it's outdated then a link would definitely work.

Reply:
Got it, let's link to https://about.mattermost.com/support/ instead of the email. I've submitted a fix for the mobile troubleshooting page https://github.com/mattermost/docs/pull/2838